dash_auth/plotly_auth.py

```diff
@@ -13,8 +13,14 @@
 from hmac import compare_digest
 from six import iteritems
 
-import dash_html_components as html
-import dash_core_components as dcc
+from distutils.version import LooseVersion
+from dash import __version__ as dash_version
+if LooseVersion(dash_version) >= LooseVersion("2"):
+    from dash import html, dcc
+else:
+    import dash_html_components as html
+    import dash_core_components as dcc
```

Comment:
Not a big deal, but elsewhere we've done this as a `try / except ImportError` rather than an explicit version number check.